### DISCUSSION TITLE
✅ No line breaks in pasted text

### DISCUSSION THREAD

**Discussion by zzrwood** [isontheline/pro.webssh.net#165] [Q&A] [answered]
Hi
When I paste text I have copied (say from Notes or Safari on my iPad) into a session window the line breaks are converted to spaces.
Am I doing something wrong or is this standard behaviour?
Thanks
Robert
Sunshine Coast, Australia

**Comment by isontheline** [comment]
Hi Robert 👋

I don't think it's the wanted behavior ☹️

I'm opening an [issue](https://github.com/isontheline/pro.webssh.net/issues/166) right now in order to check this bug.

Thank you a lot for your report Robert!

**Reply by isontheline** [reply]
I can confirm : I'm having the same issue :
![image](https://user-images.githubusercontent.com/44212923/104230673-27fce700-544e-11eb-9e1c-9ce6ba5523a9.png)

**Comment by isontheline** [comment]
I have found the issue :
![Capture d’écran 2021-01-11 à 21 24 26](https://user-images.githubusercontent.com/44212923/104234259-706ad380-5453-11eb-8205-c1388829a5d0.png)

Do you have TestFlight app in order to beta test WebSSH?

**Comment by zzrwood** [comment]
Don’t have TestFlight - but happy to get it to beta test...

**Comment by isontheline** [comment]
Thank you. I will provide you a link in a few hours when the beta release will available.

have a nice day ☀️

**Comment by isontheline** [comment]
@zzrwood  

Please install _WebSSH Beta_ 14.10.3 from TestFlight app : https://testflight.apple.com/join/QSrBK59z

**Comment by isontheline** [comment, marked as answer]
You will then be able to choose the carriage return char :
![image](https://user-images.githubusercontent.com/44212923/104279776-26add780-54ab-11eb-9215-e322b9b604b3.png)

By default it's \r **so you don't have** to change anything inside WebSSH in order to fix your issue
